aws-functions/HttpTriggerAPIUsers/lambda_function.py

```diff
@@ -0,0 +1,153 @@
+import os
```

Comment:
Thank you for setting up the folder, I will be sure to add my code to aws-functions